pkg/sql/sem/builtins/builtins.go

```diff
@@ -1702,16 +1702,18 @@ var builtins = map[string]builtinDefinition{
 			tree.VolatilityImmutable,
 		)),
 
-	"similar_to_escape": makeBuiltin(defProps(),
-		stringOverload3(
-			"unescaped", "pattern", "escape",
-			func(evalCtx *tree.EvalContext, unescaped, pattern, escape string) (tree.Datum, error) {
-				return tree.SimilarToEscape(evalCtx, unescaped, pattern, escape)
-			},
-			types.Bool,
-			"Matches `unescaped` with `pattern` using 'escape' as an escape token.",
-			tree.VolatilityImmutable,
-		)),
+	"similar_escape": makeBuiltin(
+		tree.FunctionProperties{
+			NullableArgs: true,
+		},
+		similarEscapeOverloads[0],
+		similarEscapeOverloads[1]),
+
+	"similar_to_escape": makeBuiltin(
+		defProps(),
+		similarEscapeOverloads[0],
```

Comment:
can you make similarEscapeOverloads only contain the two commonly supported, then:

```
"similar_to_escape": makeBuiltin(
   defProps(),
	stringOverload3(
		"unescaped", "pattern", "escape",
		func(evalCtx *tree.EvalContext, unescaped, pattern, escape string) (tree.Datum, error) {
			return tree.SimilarToEscape(evalCtx, unescaped, pattern, escape)
		},
		types.Bool,
		"Matches `unescaped` with `pattern` using 'escape' as an escape token.",
		tree.VolatilityImmutable,
	),
   similarOverloads...
)
```